common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -117,6 +117,9 @@
 new_contract('BlockKey', BlockKey)
 new_contract('XBlock', XBlock)
 
+# Get detached categories.
+_DETACHED_CATEGORIES = [name for name, __ in XBlock.load_tagged_classes("detached")]
```

Comment:
Its name was starting from '_'(underscore) thats why I thought it would be bad idea to import rather I may write up a public method there which will return _DETACHED_CATEGORIES global and use that method in get_items?